src/main/scala/com/phasmidsoftware/args/Args.scala

```diff
@@ -301,12 +313,12 @@ object Args {
   def create(args: Arg[String]*): Args[String] = apply(args)
 
   private def doParse(ps: Seq[PosixArg], wo: Option[String] = None): Args[String] = {
-    val so = (new SynopsisParser).parseSynopsis(wo)
+    val sy = (new SynopsisParser).parseOptionalSynopsis(wo)
 
     def processPosixArg(p: PosixArg): Seq[PosixArg] = p match {
       case PosixOptionString(w) =>
-        so match {
-          case Some(s) =>
+        sy match {
+          case Success(s) =>
```

Comment:
I like the approach of moving away from Option to Try. It will give better way to handle unseen error.